python/ray/serve/model_wrappers.py

```diff
@@ -248,6 +248,14 @@ async def predict(self, inp):
         """Perform inference directly without HTTP."""
         return await self.predict_impl(inp)
 
+    def reconfigure(
+        self,
+        checkpoint: Union[Checkpoint, str],
+        predictor_cls: Union[str, Type["Predictor"]],
```

Comment:
serve's reconfigure takes a single dictionary instead of multiple arguments. https://docs.ray.io/en/latest/serve/core-apis.html#user-configuration-experimental please wrap them in dict.

Reply:
> For the error you are getting locally. I recommend deleting the `ray/python/ray/thirdparty_files` directory and run `pip install -e . --verbose` in `ray/python` directory again. This happens because you compiled ray with one version but switched to another environment to run the test.

Followed instructions still getting the same error. Seems like a known issue which could be resolved if I upgrade to 3.9 instead of python 3.8

Reply:
Did it work? We are still missing a reconfigure call in the test case